docs/root/configuration/access_log.rst

```diff
@@ -296,6 +296,19 @@ The following command operators are supported:
   TCP
     Not implemented ("-").
 
+%REQ_WITHOUT_QUERY(X?Y):Z%
+  HTTP
+    Same as **%REQ(X?Y):Z%** but it removes query string from a header entry value.
+    For example, for the following header entry:
+
+    ``":path" : "/?ok=true"``
+
+    * ``%REQ(:PATH)%`` will log: ``/?ok=true``
+    * ``%REQ_WITHOUT_QUERY(:PATH)%`` will log: ``/``
```

Comment:
will this ever be used with any other header? wondering if it makes more sense to go with the other suggestion in the issue with making this a top level format

Reply:
OH, I can also see: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer that's why I went ahead with this. But yeah, probably a suffix operator is better?

`REQ(X?Y):<MAX_LENGTH>:<POST_PROCESSING_OPERATOR>`, e.g. `REQ(:PATH):2:REMOVE_QUERY, REQ(:PATH):-:REMOVE_QUERY`. Need to go back to discuss this in the issue then.

Reply:
As long as there are more than one header this makes sense for I'm fine with this. Are suffix operators something we currently support? I don't feel super strongly about either, would probably go with whichever has precedent